service/version_bundles.go

```diff
@@ -4,7 +4,7 @@ import (
 	"github.com/giantswarm/versionbundle"
 
 	awsv22 "github.com/giantswarm/cluster-operator/service/controller/aws"
-	azurev22 "github.com/giantswarm/cluster-operator/service/controller/azure/v22"
+	azurev22 "github.com/giantswarm/cluster-operator/service/controller/azure"
```

Comment:
The alias is actually useless now. Same with AWS.

Reply:
Done.